src/store/reducer/board.ts

```diff
@@ -75,6 +83,12 @@ export const boardReducer = (state: BoardState = {status: "unknown"}, action: Re
       };
     }
     case ActionType.UpdatedBoard: {
+      // User notification
+      if (state.data?.voting !== action.board.voting && action.board.voting !== undefined) {
```

Comment:
As far as I know `action.board.voting` cannot be undefined

Reply:
Yes, you are right. I thought I had this case during testing, but can't reproduce it. I have removed this test.